test/autorest/additionalpropsgroup/zz_generated_models.go

```diff
@@ -176,12 +182,14 @@ type PetAPInPropertiesWithAPStringResponse struct {
 }
 
 type PetAPObject struct {
-	// Contains additional key/value pairs not defined in the schema.
+	// REQUIRED
+	ID *int32 `json:"id,omitempty"`
+
+	// OPTIONAL; Contains additional key/value pairs not defined in the schema.
```

Comment:
I intentionally left `OPTIONAL` for additional properties as they are by definition optional.